en/frames_rover/ackermann_rover.md

```diff
@@ -255,3 +160,9 @@ $$
 | $r_{min}$      | Minimum turn radius                | m    |
 | $\delta_{max}$ | Maximum steer angle                | m    |
 | $r_{acc}$      | Acceptance radius                  | m    |
+
+<!--
```

Comment:
As a pattern for docs, normally we'd list ALL parameters affecting the frame that the user needs to know about in a table. Above we'd **list** or describe the items in each section, linking down to here. So for example, the mission parameters above would be a description or a list. 

The idea being that you have all the parameters in one place and link down to them from descriptive text. Something to think about. You might not want to do this if you thought moving the mission parameters to the drive mode docs made sense.

Reply:
So this comment is "something to think about"